pom.xml

```diff
@@ -9,7 +9,7 @@
   </parent>
   <groupId>org.folio</groupId>
   <artifactId>mod-calendar</artifactId>
-  <version>2.4.3-SNAPSHOT</version>
+  <version>2.6.0-SNAPSHOT</version>
```

Comment:
I haven't worked with maven much but shouldn't this be 2.5.0-SNAPSHOT since we're releasing v2.5.0

Reply:
there's a few commits here that do everything, the command `mvn release:prepare` does most of the heavy lifting behind the scenes

- first commit (manual): set it to `2.5.0-SNAPSHOT` as the WIP version of 2.5.0
- second commit (maven step): set to `2.5.0` — this is where the formal release is
- third commit (maven step): set to `2.6.0-SNAPSHOT` — prepare for WIP of future development

Reply:
however, i do need to fix this...the master merge needs to happen before these

Reply:
Oh, I see. Looks good to me